gigaumbrella/values.yaml

```diff
@@ -996,6 +996,7 @@ jenkins:
                               parameters{
                                   textParam('BRANCH_NAME', 'master', '[REQUIRED PARAMETER] The name of the branch')
                                   textParam('JAVA_BUILD_URL', '#http://hercules/$version/$branch/$build-name/gigaspaces-xap-enterprise-$build-name.zip', '[REQUIRED PARAMETER] XAP package location')
+                                  textParam('S3_BUILD_BUCKET', 'gigaspaces-releases-eu', 'Name of S3 bucket to upload the packages')
```

Comment:
there is also a [gigaspaces-releases-eu-testing](https://s3.console.aws.amazon.com/s3/buckets/gigaspaces-releases-eu-testing)

Reply:
Yes, but this PR is just so that the Tikal jenkins allow variables there